File: app/src/main/java/app/android/ttp/mikazuki/yoshinani/event/BaseEvent.java
package app.android.ttp.mikazuki.yoshinani.event;

import android.support.annotation.Nullable;

/**
 * @author haijimakazuki
 */
public class BaseEvent {

    private String mTag;

    @Nullable
    public String getTag() {
        return mTag;
    }

    public BaseEvent setTag(@Nullable final String mTag) {
        this.mTag = mTag;
        return this;
    }

}

File: app/src/main/java/app/android/ttp/mikazuki/yoshinani/event/BaseEvent.kt
package app.android.ttp.mikazuki.yoshinani.event

/**
 * @author haijimakazuki
 */
open class BaseEvent {

    private var mTag: String? = null

    fun getTag(): String? {
        return mTag
    }

    fun setTag(mTag: String?): BaseEvent {
        this.mTag = mTag
        return this
    }

}
